defi/src/adaptors/handlers/storeAdaptorData/index.ts

```diff
@@ -49,18 +49,23 @@ export type IStoreAdaptorDataHandlerEvent = {
   maxConcurrency?: number
   isDryRun?: boolean
   isRunFromRefillScript?: boolean
+  yesterdayIdSet?: Set<string>
+  todayIdSet?: Set<string>
+  runType?: 'store-all' | 'default'
 }
 
 const ONE_DAY_IN_SECONDS = 24 * 60 * 60
 
 export const handler2 = async (event: IStoreAdaptorDataHandlerEvent) => {
   const defaultMaxConcurrency = 21
-  let { timestamp, adapterType, protocolNames, maxConcurrency = defaultMaxConcurrency, isDryRun = false, isRunFromRefillScript = false, } = event
+  let { timestamp, adapterType, protocolNames, maxConcurrency = defaultMaxConcurrency, isDryRun = false, isRunFromRefillScript = false,
+    runType = 'default', yesterdayIdSet = new Set(), todayIdSet = new Set(),
+
+  } = event
   if (!isRunFromRefillScript)
     console.info(`- Date: ${new Date(timestamp! * 1e3).toDateString()} (timestamp ${timestamp})`)
   // Timestamp to query, defaults current timestamp - 2 minutes delay
-  const isTimestampProvided = timestamp !== undefined
-  const currentTimestamp = timestamp ?? LAMBDA_TIMESTAMP;
+  const currentTimestamp = timestamp ?? timestampAnHourAgo;
   // Get clean day
   let toTimestamp = getTimestampAtStartOfDayUTC(currentTimestamp)
```

Comment:
this code always looked sus to me since this should only be used for v1

Reply:
this is checked and updated per adapter level, think I set a default here to preemptively pull and cache chain blocks